### PR TITLE
Use transpiled cozy-ui

### PR DESCRIPTION
**Pull request by gregorylegarec**
This PR applies guidelines from https://github.com/cozy/cozy-ui/blob/master/docs/how-to-transpile.md#how-to

### CODE REVIEW

File: app.config.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 const configs = [
   require('cozy-scripts/config/webpack.bundle.default'),
   require('cozy-scripts/config/webpack.config.css-modules'),
-  require('./config/webpack.config.piwik')
+  require('./config/webpack.config.piwik'),
+  require('./config/webpack.config.cozy-ui')
 ]
```
Comment (CPatchane) [inline app.config.js]: Why not using an alias here instead of adding `/transpiled` everywhere?

File: app.config.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 const configs = [
   require('cozy-scripts/config/webpack.bundle.default'),
   require('cozy-scripts/config/webpack.config.css-modules'),
-  require('./config/webpack.config.piwik')
+  require('./config/webpack.config.piwik'),
+  require('./config/webpack.config.cozy-ui')
```
Comment (edas) [inline app.config.js]: This file seems missing in the PR

File: app.config.js (hunk 1 above)
Comment (edas) [reply app.config.js]: If an alias is working, that's also fine with me a easy to test/remove later

File: app.config.js (hunk 2 above)
Comment (gregorylegarec) [reply app.config.js]: Classic, thanks !

File: src/components/App.jsx [hunk 3]
```diff
@@ -2,13 +2,15 @@ import React, { Component } from 'react'
...
+
+import 'cozy-ui/transpiled/react/stylesheet.css'
+import { IconSprite } from 'cozy-ui/transpiled/react'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import { Layout, Main } from 'cozy-ui/transpiled/react/Layout'
 import { Route, Switch, Redirect, withRouter } from 'react-router-dom'
 
 import Sidebar from 'components/Sidebar'
-import Alerter from 'cozy-ui/react/Alerter'
+import Alerter from 'cozy-ui/transpiled/react/Alerter'
```
Comment (edas) [inline src/components/App.jsx]: Did you forgot to update this file and remove the "transpiled" in it ?

Comment (gregorylegarec) [reply src/components/App.jsx]: Probably. We cannot trust Search and replace :(

Comment (gregorylegarec) [reply src/components/App.jsx]: Thanks !

File: src/components/App.jsx [hunk 4]
```diff
@@ -2,8 +2,10 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { hot } from 'react-hot-loader'
+
+import 'cozy-ui/react/stylesheet.css'
+import { IconSprite } from 'cozy-ui/react'
```
Comment (edas) [inline src/components/App.jsx]: Je suis gêné par cette modif. Au minimum elle est indépendante de cette PR, non ?

Comment (gregorylegarec) [reply src/components/App.jsx]: C'est l'application du point 3.

**Pull request merged**